sklearn/linear_model/_base.py

```diff
@@ -478,7 +479,7 @@ def __init__(self, *, fit_intercept=True, normalize=False, copy_X=True,
         self.copy_X = copy_X
         self.n_jobs = n_jobs
 
-    def fit(self, X, y, sample_weight=None):
+    def fit(self, X, y, sample_weight=None, cholesky=False):
```

Comment:
This needs to be a parameter to ``__init__``, not ``fit``, and needs to be documented there. It should probably be called ``solver`` and should reflect the options in ``Ridge`` as much as possible.
Thanks!

Reply:
Done!